BlobStorage/Advanced.cs

```diff
@@ -1401,26 +1276,27 @@ private static async Task CreateSequentiallyNamedBlockBlobsAsync(CloudBlobContai
                     blobName = i.ToString("00000") + ".txt";
 
                     // Get a reference to the blob.
-                    blob = container.GetBlockBlobReference(blobName);
-
-                    // Set a property on the blob.
-                    blob.Properties.ContentType = "text/html";
+                    blob = container.GetBlobClient(blobName);
 
-                    // Set some metadata on the blob.
-                    blob.Metadata.Add("DateCreated", DateTime.UtcNow.ToLongDateString());
-                    blob.Metadata.Add("TimeCreated", DateTime.UtcNow.ToLongTimeString());
+                    Dictionary<string, string> metadata = new Dictionary<string, string>();
 
                     // Write the name of the blob to its contents as well.
                     msWrite = new MemoryStream(Encoding.UTF8.GetBytes("This is blob " + blobName + "."));
                     msWrite.Position = 0;
                     using (msWrite)
                     {
                         // Uploading the blob sets the properties and metadata on the new blob.
-                        await blob.UploadFromStreamAsync(msWrite);
+                        await blob.UploadAsync(msWrite);
                     }
```

Comment:
Use the `UploadAsync` that takes a `BinaryData.FromString("")` instead of all the `Stream` code.  We should do that for all tiny blob uploads so I'll stop leaving that comment everywhere in the PR.